src/SegmentButtonExpand/types.ts

```diff
@@ -1,8 +1,9 @@
-export interface SegmentButtonExpandProps {
+export interface ISegmentButtonExpandProps {
 	onClick: () => void;
 	isOpen: boolean;
-	filterActionCaption: string;
-	filterActionClick: () => void;
+	isHideButton?: boolean;
```

Comment:
```suggestion
	showButton?: boolean;
```
Соответственно, нужно инвертировать
И проверять, что если не передано, то чувак пользуется старым API (не использует этот prop) и надо показывать.

Reply:
Или можно просто переименовать в `hidden`